fidelius-service/src/main/java/org/finra/fidelius/services/CredentialsService.java

```diff
@@ -346,6 +353,39 @@ public Credential createCredential(Credential credential) {
         // Add the new credential
         return putCredential(credential);
     }
+    /**
+     * Get calls the rotation endpoint for a credential
+     *
+     * @param account     AWS Account alias used to look for account information
+     * @param sourceType  Source type of the credential
+     * @param source      Source of the credential
+     * @param shortKey    Short key or name associated with credential
+     * @param component   Optional component associated with credential
+     * @param region      AWS Region associated with AWS Account
+     * @param application Key representing membership section of key
+     * @param environment Key representing the environment in which the credential is associated with
+     * @return Secret
+     */
+    //@PreAuthorize("@fideliusRoleService.isAuthorized(#application, #account)")
```

Comment:
Do we need this? How is authorisation handled for rotate?

Reply:
Ah we do, that was disabled for local testing. I'll fix that